[mgmt] resources release (#34764)
Azure/azure-sdk-for-js: sdk/resources/arm-resources/CHANGELOG.md

# Release History
    
## 7.0.0 (2025-06-10)
    

> - operation groups `DeploymentOperations` and `Deployments` are split in this version. If you want to use these two operations, please use [`@azure/arm-resourcesdeployment`](https://www.npmjs.com/package/@azure/arm-resourcesdeployments/v/1.0.0-beta.1).

Reviewer: ```suggestion
- operation groups `DeploymentOperations` and `Deployments` has been split into an independent library [`@azure/arm-resourcesdeployment`](https://www.npmjs.com/package/@azure/arm-resourcesdeployments/v/1.0.0-beta.1).
```
